vtds_cluster_kvm/private/scripts/cluster_common.py

```diff
@@ -0,0 +1,731 @@
+#! python
```

Comment:
REVIEWER: this file is new and contains common library functions pulled from the original deployment script that serve all of the cluster deployment libraries.